LeetcodeProblems/Algorithms/2Sum.js

```diff
@@ -28,16 +28,26 @@ Output: [0,1]
  * @param {number} target
  * @return {number[]}
  */
- var twoSum = function(nums, target) {
-  let map ={};
-  for(let i=0;i<nums.length;i++){
-      const sum = target-nums[i];
-      if(map[parseInt(sum)] != 0){
-          return [map[sum], i];
-      } else{
-          map[nums[i]] = i;
+var twoSum = function (nums, target) {
+  let map = {};
+  for (let i = 0; i < nums.length; i++) {
+    const sum = target - nums[i];
+    if (map[parseInt(sum)] != 0) {
+      return [map[sum], i];
+    } else {
+      map[nums[i]] = i;
+    }
+  }
+
+  //Another method
+
+  for (let i = 0; i < nums.length; i++) {
+    for (let j = i + 1; j < nums.length; i++) {
+      if (nums[1] + nums[j] === target) {
+        return [i, j];
       }
+    }
   }
 };
```

Comment:
Could you encapsulate the new solution into a new method named `twoSum2`?

There are other examples in other files like `CoinChange`  https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/blob/08563058f7e93ce1688956cc26d5dc2a98b36d1a/LeetcodeProblems/Algorithms/Coin_Change.js.

Also, could you run the tests for both functions `twoSum` and `twoSum2`?

Reply:
Alright.. i would fix that